dhall-try/index.html

```diff
@@ -242,30 +263,26 @@ <h1 class="display-4">Turing-completeness is not a feature</h1>
       to do so as a specification bug.</p>
       <a href="https://github.com/dhall-lang/dhall-lang/wiki/Safety-guarantees"class="btn btn-lg btn-outline-dark bg-light">Safety Guarantees<i class="fas fa-shield-alt"></i></a>
     </div>
-    <div class="container jumbotron" style="background-color: #cfebfb">
-      <h1 class="display-4">Integration friendly</h1>
-      <p class="lead">Dhall provides a smooth migration path for mature enterprise deployments.</p>
-      <hr class="my-4">
-      <p>You can convert both ways between Dhall and JSON/YAML or read Dhall
-      configuration files directly into a language that supports a native
-      language binding.</p>
-      <a href="https://github.com/dhall-lang/dhall-lang/wiki/How-to-integrate-Dhall" class="btn btn-lg btn-outline-dark bg-light">Integrations <i class="fas fa-cogs" ></i></a>
-    </div>
     <div class="container jumbotron" style="background-color: #fbf3c8">
-      <h1 class="display-4">Tired of YAML?</h1>
-      <p class="lead">Dhall eliminates the YAML quirks that software engineers
-      dread.  The language standard learns from the mistakes of the past.</p>
+      <h1 class="display-4">Use a real programming language</h1>
+      <p class="lead">Tired of being a "YAML engineer"?  Migrate your existing
+      YAML (or JSON) files to Dhall
```

Comment:
We should consider different copy for this title and lede. Implying that YAML is not a real programming language and memeing on people that write a lot of YAML isn't great, for many reasons.

This section seems to be about validating configuration. Maybe we can say something about that instead?

Reply:
@joneshf: How about:

> Use programming language features
>
> Hold your configuration files to the same standard of quality as the rest of your code

Reply:
Thanks!